brushfire-core/src/main/scala/com/stripe/brushfire/Tree.scala

```diff
@@ -1,14 +1,14 @@
 package com.stripe.brushfire
 
+import com.stripe.bonsai.FullBinaryTreeOps
 import com.twitter.algebird._
 
-// type Tree[K, V, T] = AnnotatedTree[K, V, T, Unit]
```

Comment:
I can add it back in -- it didn't seem like it was being used (I thought it was originally commented out but maybe I was wrong). I don't feel strongly about this.